Vagrantfile

```diff
@@ -10,8 +10,11 @@ apt-get install -y ca-certificates apt-transport-https
 
 echo >/etc/apt/sources.list.d/weasyl.list \
     'deb http://apt.weasyldev.com/repos/apt/debian jessie main'
+echo >/etc/apt/sources.list.d/postgresql.list \
+    'deb http://apt.postgresql.org/pub/repos/apt/ trusty-pgdg main 9.5'
```

Comment:
This line should be 'deb https://apt.postgresql.org/pub/repos/apt/ jessie-pgdg main' (jessie, not trusty, ant https).